.github/workflows/clair-in-ci.db.yaml

```diff
@@ -0,0 +1,73 @@
+name: Clair in CI DB - test
+
+# This workflow tests latest database version of clair-in-ci and
+# pushes it to app-studio registry in case all tests passed
+
+on:
+  repository_dispatch:
+    types: trigger_workflow
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: clair-test-build
+  LATEST_TAG: latest
+  PREVIOUS_TAG: previous
+  TO_TEST: ${{ github.event.client_payload.image_tag }} # required, fail if not specified
+
+jobs:
+  test-and-push:
+    name: Test and push image
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check required parameters
+        run: |
+          if [ -z "${{ env.TO_TEST }}" ]; then
```

Comment:
and `client_payload.event`?

Reply:
True. Added - if: ${{ github.event.client_payload.event != 'pull_request' }}

Reply:
I meant to validate if `event` param itself is present :D

Reply:
> if: ${{ github.event.client_payload.event != 'pull_request' }}
I don't understand why this, image has to be tested in every kind of workflow

Reply:
Sorry, updated.